base/path.jl

```diff
@@ -124,36 +124,31 @@ normpath(a::AbstractString, b::AbstractString...) = normpath(joinpath(a,b...))
 abspath(a::AbstractString) = normpath(isabspath(a) ? a : joinpath(pwd(),a))
 abspath(a::AbstractString, b::AbstractString...) = abspath(joinpath(a,b...))
 
-@windows_only realpath(path::AbstractString) = realpath(utf16(path))
-@windows_only function realpath(path::UTF16String)
-    p::UInt32 = sizeof(path)>>1
+@windows_only function realpath(path::AbstractString)
+    path = cwstring(path)
```

Comment:
not type-stable (`path` will get `::Any`, I think?).  If we care, just rename `path` to `wpath`.  Similarly below.